python/ray/util/scheduling_strategies.py

```diff
@@ -40,6 +40,33 @@ def __init__(
         self.placement_group_capture_child_tasks = placement_group_capture_child_tasks
 
 
+@PublicAPI(stability="beta")
+class NodeSchedulingStrategy:
+    """Static scheduling strategy used to run a task or actor
+       on a particular node.
+
+    Attributes:
+        node_id: the hex id of the node where the task or actor should run.
```

Comment:
OOC how will this play with https://github.com/ray-project/ray/pull/23543 which would provide a much friendlier way of scheduling on a specific node?

Reply:
not only is it not clear if a node name is available, there is no guarantee a node name is unique. 

I think we should just provide a helper function in the SDK: get_node_id_from_node_name(str). It can lookup the gcs node table via the gcs accessor.

Reply:
Yes, we can provide a helper function for UX.